Model/Transaction.php

```diff
@@ -114,17 +114,64 @@ public function getChildTransactions()
     }
 
     /**
-     * Returns true if transaction's payment method is "wiretransfer"
+     * Returns true if transaction's xml payment method is "wiretransfer"
      *
      * @return bool
      *
      * @since 1.3.0
      */
     public function isPoiPiaPaymentMethod()
+    {
+        return self::_getTransactionXmlPaymentMethod() === BasePoiPiaPaymentMethod::PAYMENT_METHOD_WIRETRANSFER;
+    }
+
+    /**
+     * Returns payment method from transaction xml object
+     *
+     * @return string
+     *
+     * @since 1.3.0
+     */
+    private function _getTransactionXmlPaymentMethod()
     {
         $oXml = simplexml_load_string($this->getResponseXML());
-        $sPaymentId = (string) $oXml->{'payment-methods'}->{'payment-method'}['name'];
-        return $sPaymentId === BasePoiPiaPaymentMethod::PAYMENT_METHOD_WIRETRANSFER;
+        return (string) $oXml->{'payment-methods'}->{'payment-method'}['name'];
+    }
+
+    /**
+     * Returns payment method name by transaction order reference if there is order reference.
```

Comment:
Please adjust to: "Returns payment method name from transaction order reference if existing."

Reply:
👍